azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/localuser/LocalParticipantView.kt

```diff
@@ -131,6 +132,14 @@ internal class LocalParticipantView : ConstraintLayout {
         }
     }
 
+    private fun setupAccessibility() {
```

Comment:
generic name (e.g. consider applyCameraContentDescription) or something more descriptive.

Reply:
This naming was here before. Conflict resolve made it look like its this PR's changes. But I'm noting down this point and will bring up in the next refactoring meeting.